packages/dev/core/src/AudioV2/abstractAudio/staticSound.ts

```diff
@@ -0,0 +1,238 @@
+ *
+ * To prevent downloading and decoding a sound multiple times, a sound's buffer can be shared with other sounds.
+ * See {@link CreateSoundBufferAsync}, {@link StaticSoundBuffer} and {@link StaticSound.buffer} for more information.
+ *
+ * Static sounds are created by the {@link CreateSoundAsync} function.
+ */
+export abstract class StaticSound extends AbstractSound {
+    protected override _instances: Set<_StaticSoundInstance>;
+    protected override _options: IStaticSoundOptions;
+
+    /**
+     * The sound buffer that the sound uses.
+     *
+     * This buffer can be shared with other static sounds.
+     */
+    public abstract readonly buffer: StaticSoundBuffer;
+
+    protected constructor(name: string, engine: AudioEngineV2, options: Partial<IStaticSoundOptions> = {}) {
+        super(name, engine, options);
+
+        this._options.duration ??= 0;
+        this._options.loopStart ??= 0;
+        this._options.loopEnd ??= 0;
+        this._options.pitch ??= 0;
+        this._options.playbackRate ??= 1;
+        this._options.skipCodecCheck ??= false;
+    }
+
+    /**
+     * The amount of time to play the sound for, in seconds.
+     * - If less than or equal to `0`, the sound plays for its full duration.
+     */
+    public get duration(): number {
+        return this._options.duration;
+    }
+
+    public set duration(value: number) {
+        this._options.duration = value;
+    }
+
+    /**
+     * The start of the loop range, in seconds. Defaults to `0`.
+     * - If less than or equal to `0`, the loop starts at the beginning of the sound.
+     */
+    public get loopStart(): number {
+        return this._options.loopStart;
+    }
+
+    public set loopStart(value: number) {
+        this._options.loopStart = value;
+    }
+
+    /**
+     * The end of the loop range, in seconds. Defaults to `0`.
+     * - If less than or equal to `0`, the loop plays for the sound's full duration.
+     */
+    public get loopEnd(): number {
+        return this._options.loopEnd;
+    }
+
+    public set loopEnd(value: number) {
+        this._options.loopEnd = value;
+    }
+
+    /**
+     * The pitch of the sound, in cents.
+     * - Gets combined with {@link playbackRate} to determine the final pitch.
+     */
+    public get pitch(): number {
+        return this._options.pitch;
+    }
+
+    public set pitch(value: number) {
+        this._options.pitch = value;
+    }
+
+    /**
+     * The playback rate of the sound.
+     * - Gets combined with {@link pitch} to determine the final playback rate.
+     */
+    public get playbackRate(): number {
+        return this._options.playbackRate;
+    }
+
+    public set playbackRate(value: number) {
+        this._options.playbackRate = value;
+    }
+
+    protected abstract override _createInstance(): _StaticSoundInstance;
+
+    // Inherits doc from AbstractSound.
+    // eslint-disable-next-line babylonjs/available
```

Comment:
no need to document that or add the linter line, though eslint will still complain. The TypeDoc check will not complain for sure :-)